Formula/utopiops.rb

```diff
@@ -0,0 +1,14 @@
+class Utopiops < Formula
+  desc "CLI tool for utopiops.com"
+  homepage "https://www.utopiops.com/"
+  url "https://github.com/utopiops/homebrew-utopiops/releases/download/fresh/utopiops-0.0.3.tar.gz"
```

Comment:
This is a pre-built binary, which isn't allowed in Homebrew/core. We should build this from source instead. See https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae.

Reply:
Thanks, but my source code is not open source so what should I do?

Reply:
Sorry, but we only package open source software in Homebrew/core: https://docs.brew.sh/Formula-Cookbook#fill-in-the-license

Reply:
Thanks for your time.